heat/core/statistics.py

```diff
@@ -752,74 +808,10 @@ def reduce_means_elementwise(output_shape_i):
-    if isinstance(axis, (list, tuple, dndarray.DNDarray, torch.Tensor)):
-        if isinstance(axis, (list, tuple)):
-            if len(set(axis)) != len(axis):
-                raise ValueError("duplicate value in axis")
-        if isinstance(axis, (dndarray.DNDarray, torch.Tensor)):
-            if axis.unique().numel() != axis.numel():
-                raise ValueError("duplicate value in axis")
-        if any([not isinstance(j, int) for j in axis]):
-            raise ValueError(
-                "items in axis iterable must be integers, axes: {}".format([type(q) for q in axis])
-            )
-        if any(d < 0 for d in axis):
-            axis = [stride_tricks.sanitize_axis(x.shape, j) for j in axis]
-        if any(d > len(x.shape) for d in axis):
-            raise ValueError(
-                "axes (axis) must be < {}, currently are {}".format(len(x.shape), axis)
-            )
 
-        output_shape = [output_shape[it] for it in range(len(output_shape)) if it not in axis]
-        # multiple dimensions
-        if x.split is None:
-            return factories.array(
-                torch.mean(x._DNDarray__array, dim=axis), is_split=x.split, device=x.device
-            )
-
-        if x.split in axis:
-            # merge in the direction of the split
-            return reduce_means_elementwise(output_shape)
-        else:
-            # multiple dimensions which does *not* include the split axis
-            # combine along the split axis
-            return factories.array(
-                torch.mean(x._DNDarray__array, dim=axis),
-                is_split=x.split if x.split < len(output_shape) else len(output_shape) - 1,
-                device=x.device,
-            )
-    elif isinstance(axis, int):
-        if axis >= len(x.shape):
-            raise ValueError("axis (axis) must be < {}, currently is {}".format(len(x.shape), axis))
-        axis = stride_tricks.sanitize_axis(x.shape, axis=axis)
-
-        # only one axis given
-        output_shape = [output_shape[it] for it in range(len(output_shape)) if it != axis]
-        output_shape = output_shape if output_shape else (1,)
-
-        if x.split is None:
-
-            return factories.array(
-                torch.mean(x._DNDarray__array, dim=axis), is_split=None, device=x.device
-            )
-        elif axis == x.split:
-            return reduce_means_elementwise(output_shape)
-        else:
-            # singular axis given (axis) not equal to split direction (x.split)
-            return factories.array(
-                torch.mean(x._DNDarray__array, dim=axis),
-                is_split=x.split if axis > x.split else x.split - 1,
-                device=x.device,
-            )
-    raise TypeError(
-        "axis (axis) must be an int or a list, ht.DNDarray, "
-        "torch.Tensor, or tuple, but was {}".format(type(axis))
-    )
-
-
-def __merge_moments(m1, m2, bessel=True):
+def __merge_moments(m1, m2, unbiased=True):
```

Comment:
Can we have parameter and return type annotations here (except DNDarray due to aliasing)